docs/modules/examples/pages/hotels-app.adoc

```diff
@@ -1,10 +1,10 @@
-= Build a Hotel Search Application with RAGStack and AstraDB
+= Build a Hotel Search Application with RAGStack and Astra DB
```

Comment:
Should we specify Astra DB Serverless here?

Reply:
Yes.